src/js/views/Dashboard/ActivityFeed/Feed.jsx

```diff
@@ -53,15 +72,36 @@ function Feed({ onReady }) {
           {state.errorMessage !== null && (
             <Alert level="error">{state.errorMessage}</Alert>
           )}
-          <div className="h-full overflow-y-scroll">
-            <ul className="space-y-3">
-              {state.data
-                .filter((e) => e.display_name !== 'SonarQube')
-                .map((entry, offset) => {
-                  return <Entry key={`entry-${offset}`} entry={entry} />
-                })}
-            </ul>
-          </div>
+          <ul
+            id="activity-feed-list"
+            role="list"
+            className="space-y-1 h-full overflow-auto">
+            <InfiniteScroll
+              next={fetchNext}
+              hasMore={state.nextLink !== null}
+              loader={<Loading />}
+              dataLength={state.data.length}
+              scrollableTarget="activity-feed-list"
+              scrollThreshold={0.7}>
+              {state.data.map((entry, index) => {
+                let entryComponent = <></>
+                if (entry.type === 'ProjectFeedEntry')
+                  entryComponent = (
+                    <ActivityEntry key={`entry-${index}`} entry={entry} />
+                  )
+                else if (entry.type === 'OperationsLogEntry')
+                  entryComponent = (
+                    <OpsLogEntry key={`entry-${index}`} entry={entry} />
+                  )
+                return (
+                  <>
+                    {entryComponent}
+                    <div className="h-[1px] w-full bg-gray-200"></div>
+                  </>
+                )
```

Comment:
Something in here is causing a `list should have a unique "key" prop` warning ... it might be the spacer div?
```
react_devtools_backend.js:4012 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Feed`. See https://reactjs.org/link/warning-keys for more information.
    at Feed (http://localhost:8080/static/imbi.js:55807:22)
    at div
    at div
    at div
    at Dashboard (http://localhost:8080/static/imbi.js:56111:66)
    at Routes (http://localhost:8080/static/imbi.js:184537:5)
    at main
    at ErrorBoundary (http://localhost:8080/static/imbi.js:47240:5)
    at Main (http://localhost:8080/static/imbi.js:57133:19)
    at State (http://localhost:8080/static/imbi.js:54079:22)
    at App (http://localhost:8080/static/imbi.js:52749:25)
    at Router (http://localhost:8080/static/imbi.js:184471:15)
    at BrowserRouter (http://localhost:8080/static/imbi.js:183312:5)
```